src/utils/hashmap.c

```diff
@@ -160,8 +73,25 @@ int hashmap_resize(hashmap *hm, int n_size) {
     // Free old bucket list and assign new list and size of the hashmap
     omf_free(hm->buckets);
     hm->buckets = new_buckets;
-    hm->buckets_x = n_size;
-    return 0;
+    hm->capacity = new_size;
+}
+
+static void hashmap_enlarge_check(hashmap *hm) {
```

Comment:
Some comments on what these are actually checking would be nice

Reply:
Comments added.